raft/src/main/java/org/apache/kafka/raft/KafkaRaftClient.java

```diff
@@ -915,17 +939,19 @@ public void poll() throws IOException {
             pollShutdown(gracefulShutdown);
         } else {
             electionTimer.update();
+            fetchTimer.update();
```

Comment:
I think `Timer` supports a method to update to a specific time. Since we now have a couple of these, maybe it makes sense to create a method like this:
```java
void updateTimers(long currentTimeMs) {
  electionTimer.update(currentTimeMs);
  fetchTimer.update(currentTimeMs);
}
```
I guess this method could also return `Math.min(electionTimer.remainingMs(), fetchTimer.remainingMs())` to be used below.

Reply:
We added `updateTimers`, but we still have these calls to `update`. Aren't they redundant now?